src/main/resources/idprovider/idprovider.js

```diff
@@ -15,7 +27,19 @@ exports.handle401 = function() {
     };
 };
 
-exports.get = function() {
+exports.get = function (req) {
+    if (!req.rawPath.startsWith('/api/idprovider/')) {
```

Comment:
Why not use regex and .test() here as well?

Reply:
because regex is 100 slower, I guess.